### ISSUE TITLE
Logitech G502X Not Detected in Piper Although it is Supported

### ISSUE THREAD

**Issue by jaamivstheworld** [libratbag/piper#811]
**Information**

- `ratbagd` version (`ratbagd --version`): 0.16
- Piper version: 0.7
- Distribution: Fedora Workstation 37
- Device: Logitech G502X (wired)
- Kernel version (ex. `uname -srmo`): `Linux 6.0.11-300.fc37.x86_64 x86_64 GNU/Linux`

**Describe the bug**
Piper is unable to detect my Logitech G502X gaming mouse. Launching Piper gives me a screen with a mouse trap image and text saying "Cannot find any devices" and "Please make sure your device is supported and plugged in"

**To Reproduce**
Install libratbag and piper, launch piper with G502X plugged in.

1. Install libratbag-ratbagd (package name for Fedora)
2. Run ratbagd as DBus-activated systemd service
3. restart
4. Launch piper with G502X plugged 
5. See error

**Logs**
Please start the daemon with verbosity and reproduce the issue.

First make sure it isn't running already.
```
$ ratbagd --verbose=raw
Initializing libratbag
ratbag debug: New device: Logitech Gaming Keyboard G213
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c336
ratbag debug: New device: Logitech Gaming Keyboard G213
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c336
ratbag debug: New device: Logitech G502 X
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c099
ratbag debug: New device: Logitech G502 X
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c099
ratbag debug: New device: ELAN0712:00 04F3:30FD
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 04f3:30fd
DBus server ready
```

**Additional context**
No additional context. Please let me know if you need any more information.

**Comment by staticssleever668** [comment]
Fedora doesn't yet provide recently released libratbag v0.17 where the device file was added.

Until then, you can create it yourself.
Create a file named `logitech-g502-x.device` under `/usr/share/libratbag` with following contents:

```dosini
[Device]
Name=Logitech G502 X
DeviceMatch=usb:046d:c099
Driver=hidpp20
```

**Comment by PumaFPV** [comment]
Hello, my G502X wireless is not detected although it is present in /usr/share/libratbag/logitech-g502-x-wireless.device
Im on CachyOS Kernel: Linux 7.1.3-2-cachyos, ratbag 0.18 

```
❯ sudo ratbagd --verbose=raw
Starting ratbagd version 0.18 (API version 2)
Initializing libratbag
ratbag debug: New device: Logitech Logi Z407
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:0a4c
ratbag debug: New device: 8BitDo 8BitDo UM 2 Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 2dc8:3109
ratbag debug: New device: RØDE Microphones RØDE NT-USB Mini
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 19f7:0015
ratbag debug: New device: MSI MYSTIC LIGHT 
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 1462:7c37
ratbag debug: New device: AsusTek Computer Inc. ROG Gaming Display Aura Device
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0b05:1c8a
ratbag debug: New device: Realtek HID Device
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0bda:1100
ratbag debug: New device: Elgato Stream Deck XL
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0fd9:008f
ratbag debug: New device: 3Dconnexion SpaceNavigator
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c626
ratbag debug: New device: Logitech K340
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:2007
ratbag debug: New device: Logitech K270
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:4003
ratbag debug: New device: Logitech M185
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:4008
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: 046d:c52b is a Logitech receiver, not a device. Ignoring...
ratbag debug: New device: PumaCorp PMK Dongle
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c62b
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: 046d:c547 is a Logitech receiver, not a device. Ignoring...
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: 046d:c547 is a Logitech receiver, not a device. Ignoring...
ratbag debug: New device: Logitech G502 X PLUS
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:4099
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: 046d:c547 is a Logitech receiver, not a device. Ignoring...
DBus server ready
```
